doc-gen/Makefile

```diff
@@ -1,10 +1,2 @@
 all:
 	go run . `tinygo targets`
-
-
-#all:
-#	go run . pca10040 > ../content/doc/pca10040/machine.md
-#	go run . hifive1b > ../content/doc/hifive1b/machine.md
-#	go run . bluepill > ../content/doc/bluepill/machine.md
-#	go run . gameboy-advance > ../content/doc/gameboy-advance/machine.md
-#
```

Comment:
Heh, I probably forgot to remove this at some point.

Reply:
:smile_cat: